### PR TITLE
allow pushes with options (scheduled, delay)

### PR DESCRIPTION
**Pull request by BnitoBzh**
allow pushes with options (scheduled, delay)

### CODE REVIEW

File: src/Controller/Plugin/QueuePlugin.php [hunk 1]
```diff
@@ -73,10 +73,11 @@ public function __invoke($name = null)
      *
      * @param  string $name    Name of the job to create
      * @param  mixed $payload  Payload of the job set as content
+     * @param  array $options  Push action options, ie. scheduled, delay
```
Comment (basz) [inline src/Controller/Plugin/QueuePlugin.php]: `Push action options` change to `Push job options`

**Comment by BnitoBzh** [comment]
tests added

**Comment by BnitoBzh** [comment]
I have updated my tests, because "scheduled" and "delay" options are specific data (used on DoctrineQueue but not on SqsQueue)

**Comment by basz** [comment]
thanks, i've release 0.8 with only this addition

**Comment by BnitoBzh** [comment]
Is it possible to release to 0.7.1 only ? not 0.8, because there is a conflict with the slmqueuedoctrine module requirements, or change the requirements to ^0.7 || ^0.8 ?

**Comment by basz** [comment]
I think we shuold release 1.0.0 on all packages

**Comment by BnitoBzh** [comment]
Yes, it's a great idea, and clearly specify that all repos works with ZF3 in their README files.

**Comment by basz** [comment]
slm-queue-* should have depended on ~7.0 instead of ^7.0. That way slm-queue:0.8.0 would be allowed. But since we stable for a long while now I think a 1.0.0 is justifiable. cc/@bakura10

**Comment by bakura10** [comment]
+1 for a 1.0.0. I'm not using SlmQueue anymore but the 0.8 is quite solid imho. Goes on :D.

**Comment by BnitoBzh** [comment]
Hi @basz , @bakura10 , is it possible to quickly update versions for all repos ? (version 1.0.0)

**Comment by basz** [comment]
Yes, Will do tonight. Ping again if i dont

**Comment by BnitoBzh** [comment]
Ok, thanks !

**Pull request merged**